cypress/integration/customer_creation.js

```diff
@@ -0,0 +1,23 @@
+
+context('Customer', () => {
+	before(() => {
+		cy.login();
+	});
+	it('Insert a Customer ', () => {
+		cy.visit(`app/customer/`);
+		cy.get('.primary-action').click();
+		cy.get('.custom-actions > .btn').click();
+		cy.get_field('customer_group', 'Link').type("All Customer Groups");
+		cy.get_field('customer_name', 'Link').type("Lara Schindler");
+		cy.get_field('territory', 'Link').type("All Territories");
+		cy.get('#page-Customer > .page-head > .container > .row > .col > .standard-actions > .primary-action').click();
+	});
+
+	it('Check Customer attributes ', () => {
```

Comment:
This is checking the state left by the previous "test". These two tests are actually a single test.